ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -505,7 +505,7 @@ final class Metrics(val registry: MetricRegistry) {
           "get_contract_state_events"
         )
         val loadStringInterningEntries: DatabaseMetrics = createDbMetrics(
-          "loadStringInterningEntries"
+          "load_string_interning_entries"
```

Comment:
Add a change log entry so that the release notes can mention this change of name, please.